tests/web_api/__init__.py

```diff
@@ -3,5 +3,14 @@
 from openfisca_web_api_preview.app import create_app
 
 TEST_COUNTRY_PACKAGE_NAME = 'openfisca_country_template'
+TRACKER_URL = 'https://openfisca.innocraft.cloud/piwik.php'
+TRACKER_IDSITE = 1
+
+ACTIVE_TRACKING = True
+
 distribution = pkg_resources.get_distribution(TEST_COUNTRY_PACKAGE_NAME)
-subject = create_app(TEST_COUNTRY_PACKAGE_NAME).test_client()
+
+if ACTIVE_TRACKING:
+    subject = create_app(TEST_COUNTRY_PACKAGE_NAME, TRACKER_URL, TRACKER_IDSITE).test_client()
```

Comment:
Ok for removing it. It's a way to test the whole chain without writing requests. But, it's only interesting when we use a Piwik instance dedicated to tests.